draft-ietf-rtcweb-jsep.xml

```diff
@@ -3592,11 +3592,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             in an error.</t>
 
             <t>If the media section has RTCP mux enabled, discard any
-            RTCP component, and begin or continue muxing RTCP over the
-            RTP component, as specified in
+            RTCP ICE component, and begin or continue muxing RTCP over the
```

Comment:
I know this was there before "any RTCP ice component" is weird sounding. Maybe "the RTCP ICE component, if any"?